planningcenter/grids/horizontal-main-nav/stories/horizontal-main-nav--components.mdx

```diff
@@ -0,0 +1,97 @@
+import { Meta, Story, Canvas, Props } from "@storybook/addon-docs/blocks";
+
+import TabbedNavigation, * as HorizontalMainNav from "./horizontal-main-nav.tsx";
+import "./horizontal-main-nav.css";
+
+<Meta title="Grids/Horizontal Main Nav/Components/Link" />
+
+<Props of={HorizontalMainNav.Link} />
+
+<HorizontalMainNave.Link />
```

Comment:
Typo? "HorizontalMainNave"

Reply:
oops. that file got merged into another file and not removed. thanks for catching. deleted now